debugging/tasks/analyze.js

```diff
@@ -536,11 +598,25 @@ exports.run = function (extra, args) {
   };
 
   const printCollections = function (info) {
-    let table = new AsciiTable('collections');
+    const table = new AsciiTable('collections');
     table.setHeading('', 'CID', 'RF', 'Shards Like', 'Shards', 'Type', 'Smart');
 
+    const rfs = {};
+
+    _.each(info.collections, function (collection, name) {
+      if (!collection.distrbuteShardsLike) {
```

Comment:
```suggestion
      if (!collection.distributeShardsLike) {
```

This small typo breaks the algorithm here.

Reply:
only print so repair will still work not cirtical

Reply:
fixed